intro-OpenStack/ch_LaunchInstances.tex

```diff
@@ -0,0 +1,272 @@
+\chapter{Launch and manage instances}\label{cha:launch-manage-inst}
+Instances are virtual machines that run inside the cloud. You can launch
+an \gls{instance} from the following sources:
+
+\begin{itemize}
+\item Images uploaded to the Image service.  Note that, because images
+    are read-only, any changes made while the instance is running will
+    be lost after shutdown, unless you choose to ``Create New
```

Comment:
This is not 100% true, the VM will keep the changes if you shutdown or reboot the instance, that is not an issue. The changes are lost if you delete the running instance completely using the trash bin icon.